src/js/tech/tech.js

```diff
@@ -312,18 +312,18 @@ class Tech extends Component {
    * @method emulateTextTracks
    */
   emulateTextTracks() {
+    let tracks = this.textTracks();
+    if (!tracks) {
+      return;
+    }
+
     if (!window['WebVTT'] && this.el().parentNode != null) {
       let script = document.createElement('script');
-      script.src = this.options_['vtt.js'] || '../node_modules/vtt.js/dist/vtt.js';
+      script.src = this.options_['vtt.js'] || 'https://raw.githubusercontent.com/mozilla/vtt.js/v0.12.1/dist/vtt.js';
```

Comment:
Replacing on build would make sense. The built module should not have a reference to node_modules